addons/survey/static/src/scss/survey_templates_form.scss

```diff
@@ -131,14 +131,14 @@ _::-webkit-full-page-media, _:future, :root .o_survey_wrap {
         i {
             top: 0px;
             font-size: large;
-            &.fa-check-circle {
+            &.fa-check-circle, &.fa-check-square {
                 display: none;
             }
         }
 
         &.o_survey_selected i {
             display: none;
-            &.fa-check-circle {
+            &.fa-check-circle, &.fa-check-square{
```

Comment:
Tiny remark: There is a missing space before the `{` bracket.

---

:bulb: **Suggestion** (can be ignored): I think we could simplify the css a bit and gather the rules hiding the icons like so:

```scss
&.o_survey_selected i.fa-circle-thin,
&.o_survey_selected i.fa-square-o,
&:not(.o_survey_selected) i.fa-check-circle,
&:not(.o_survey_selected) i.fa-check-square {
    display: none;
}
```

With that selector, we avoid defining and overwriting our own CSS rules. We just hide the icons when it is necessary.

Reply:
Hello @julien-banken :slightly_smiling_face: 
Thanks for the validation!

I pushed a new version with your suggestion :smile: 
I kept this rule at the end of the css file though as this rule is specific to the survey print page :slightly_smiling_face: (cf selector line 448 vs selector line 69) :
```css
i.fa-circle-thin, i.fa-square-o {
    display: none;
}
```

Thanks and cheers!